pkg/admission/denypsalabel/admission.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package denypsalabel
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/apis/core"
+)
+
+const (
+	// PluginName is the name of this admission controller plugin
+	PluginName     = "DenyPSALabel"
+	PSALabelPrefix = "pod-security.kubernetes.io"
```

Comment:
nit: I don't think this needs to be exported, just use it as it is upstream. 
ref: https://github.com/kubernetes/kubernetes/blob/v1.31.1/staging/src/k8s.io/pod-security-admission/api/constants.go#L38

```suggestion
	labelPrefix = "pod-security.kubernetes.io/"
```